packages/wrangler/src/bundle.ts

```diff
@@ -60,6 +61,30 @@ export async function bundleWorker(
     format: entry.format,
     rules,
   });
+  const checkForNodeBuiltinsPlugin = {
```

Comment:
Do we have a better place to put esbuild plugins?

Reply:
It's fine inside here for now